src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityAssemblyLine.java

```diff
@@ -34,7 +42,7 @@ public MetaTileEntity createMetaTileEntity(MetaTileEntityHolder holder) {
     @Override
     protected BlockPattern createStructurePattern() {
         return FactoryBlockPattern.start(FRONT, UP, RIGHT)
-                .aisle("FIF", "RTR", "SAG", "#Y#")
+                .aisle("FIF", "RTR", "SAD", "#Y#")
```

Comment:
The data hatch should be allowed on any grate casing, this should be changed back to `SAG`.